src/core_plugins/kibana/public/dashboard/index.html

```diff
@@ -56,7 +53,7 @@
 
   <div ng-show="showEditHelpText()" class="text-center start-screen">
     <h2>Ready to get started?</h2>
-    <p>Click the <a class="btn btn-xs navbtn-inverse" ng-click="kbnTopNav.open('add'); toggleAddVisualization = !toggleAddVisualization" aria-label="Add visualization">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't setup a visualization yet visit the <a href="#/visualize" title="Visualize">"Visualize"</a> tab to create your first visualization.</p>
```

Comment:
Re Uri's comment, how about changing the title to: "This dashboard is empty. Let's fill it up!"